handbook/engineering/search/iteration_log.md

```diff
@@ -14,9 +14,41 @@ This document contains the goals and work log for the search team's [2-week iter
 
 ## 2020-09-21 to 2020-10-02
 
-### $GOAL_OR_THEME
+### Scale indexed search to 500k repositories
 
-- **Owner(s):** $OWNER
+- **Owner(s):** Stefan
+- **Outcomes:**
+    - $DESIRED_OUTCOME
+- **Work log:**
+    - YYYY-MM-DD: $UPDATE
+
+### Streaming search
+
+- **Owner(s):** Keegan
+- **Outcomes:**
+    - $DESIRED_OUTCOME
+- **Work log:**
+    - YYYY-MM-DD: $UPDATE
+
+### Enterprise homepage
+
+- **Owner(s):** Juliana
+- **Outcomes:**
+    - $DESIRED_OUTCOME
```

Comment:
@limitedmage can you fill in the desired outcome(s) for the enterprise homepage project for the next iteration?

Reply:
Does this sound reasonable?

```suggestion
    - QA issues with the enterprise homepage are discovered and resolved
    - Once we feel the feature has the right level of polish, the feature flag is turned on by default so that Sourcegraph Server users see it in 3.21
```

Reply:
Maybe we can be a bit more specific. How do you feel about this?

```suggestion
    - The feedback resulting from [design QA](https://www.figma.com/file/sPRyyv3nt5h0284nqEuAXE/%2312192-Sourcegraph-server-page-v1?node-id=2274%3A21539) has been implemented.
    - The feature has been QA'd internally.
    - If all feedback resulting from internal QA can be integrated during the iteration, the feature flag is turned on by default.
```